WebServer/form.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>HELLO</title>
+<body>
+<br><br>
+<center>
+    <form action="/server_first.py" method="post">
```

Comment:
I think your `action` should be `/charge.html` here, right?

Reply:
Yes, it is corrected in Task2, i'll fix it here too.

Reply:
Corrected